artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -2953,6 +2961,15 @@
                </xsd:annotation>
             </xsd:element>
 
+            <xsd:element name="auto-delete-queues-delay" type="xsd:long" default="0" maxOccurs="1" minOccurs="0">
```

Comment:
ditto - Can this be added to the artemis-configuration used for testing to keep as much as possible the two in sync